extensions/pathwatcher/internal.m

```diff
@@ -72,6 +72,8 @@ void event_callback(ConstFSEventStreamRef __unused streamRef, void *clientCallBa
 ///   * ownEvent (OS X 10.9+)
 ///   * itemIsHardlink (OS X 10.10+)
 ///   * itemIsLastHardlink (OS X 10.10+)
+///
+/// See [the official documentation](https://developer.apple.com/reference/coreservices/1455361-fseventstreameventflags/kfseventstreameventflagitemrenamed?language=objc) for details.
```

Comment:
I think you probably meant to link to this instead?

https://developer.apple.com/reference/coreservices/1455361-fseventstreameventflags/